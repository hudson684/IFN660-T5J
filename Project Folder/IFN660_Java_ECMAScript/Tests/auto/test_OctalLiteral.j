2
032
0l
0777L
0_52
05_2
0_5_2_3_4
07_7777_7777_7777_7777_7777l
07_7777_7777_7777_7777_7777L
010_0000_0000_0000_0000_0000l
010_0000_0000_0000_0000_0000L