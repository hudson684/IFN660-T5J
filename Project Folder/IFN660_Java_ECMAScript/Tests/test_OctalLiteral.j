valid
0
2
032
0l
0777L
0_52
05_2
0_5_2_3_4
07_7777_7777_7777_7777_7777l
07_7777_7777_7777_7777_7777L
010_0000_0000_0000_0000_0000l
010_0000_0000_0000_0000_0000L

invalid
052_
0_5_2_
0052__
0__52
_0_52