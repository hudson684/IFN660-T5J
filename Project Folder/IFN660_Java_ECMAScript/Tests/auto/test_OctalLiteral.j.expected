IntegerLiteral (2)
IntegerLiteral (26)
IntegerLiteral (0)
IntegerLiteral (511)
IntegerLiteral (0)
IntegerLiteral (42)
IntegerLiteral (42)
IntegerLiteral (0)
IntegerLiteral (2716)
IntegerLiteral (9223372036854775807)
IntegerLiteral (9223372036854775807)
IntegerLiteral (-9223372036854775808)
IntegerLiteral (-9223372036854775808)
EOF